xbmc/GUIInfoManager.cpp

```diff
@@ -6227,6 +6239,12 @@ std::string CGUIInfoManager::GetLabel(int info, int contextWindow, std::string *
   case SYSTEM_SCREEN_HEIGHT:
     strLabel = StringUtils::Format("%i", g_graphicsContext.GetResInfo().iScreenHeight);
     break;
+  case SYSTEM_MOUSE_POSX:
+    strLabel = StringUtils::Format("%i", GetMousePosition()[0]);
+    break;
+  case SYSTEM_MOUSE_POSY:
+    strLabel = StringUtils::Format("%i", GetMousePosition()[1]);
```

Comment:
I'd rather `GetMousePosition()` return a `CPoint` so that we could do: `GetMousePosition().x`. Might be slightly more clear than [0].

That way, `CGUIWindowDebugInfo` can continue processing a `CPoint`, and the conversion to `std::vector<int>` can happen in ModuleXbmc.cpp